integration-tests/reactive-messaging-kafka/src/main/resources/application.properties

```diff
@@ -0,0 +1,14 @@
+quarkus.log.category.kafka.level=WARN
+quarkus.log.category.\"org.apache.kafka\".level=WARN
+quarkus.log.category.\"org.apache.zookeeper\".level=WARN
+
+# enable health check
+quarkus.kafka.health.enabled=true
+kafka.bootstrap.servers=localhost:19092
+
+mp.messaging.outgoing.people-out.connector=smallrye-kafka
+mp.messaging.outgoing.people-out.topic=people
+mp.messaging.outgoing.people-out.value.serializer=io.quarkus.kafka.client.serialization.JsonbSerializer
+mp.messaging.incoming.people-in.connector=smallrye-kafka
+mp.messaging.incoming.people-in.topic=people
```

Comment:
The tests fail because of a missing property:
```
mp.messaging.incoming.people-in.auto.offset.reset=earliest
```

Without, it uses Kafka's. default which is _latest_. Basically, it only receives records sends after the subscription. _earliest_ instructs the consumers to _reset_ its offset to the last processed offset from consumers from its consumer group. Here, it will reset the offset to 0, as there are no other consumers (so obviously none in its group) and no records have been erased due to retention expiration (24h by default).

Reply:
Indeed, many thanks @cescoffier for the help and the detailed explanation as well.